lib/yardstick/rule_description/formatter.rb

```diff
@@ -0,0 +1,47 @@
+module Yardstick
+  class RuleDescription
+    # Rule description text formatter
+    class Formatter
+      include Adamantium, Concord.new(:description)
+
+      # Map of token types to decorators
+      TOKEN_DECORATORS = {
```

Comment:
I noticed you removed the `IceNine.deep_freeze` from here. Is that because it was deep freezing the class itself?